src/aws/models/citadel-rewards-snapshot.ts

```diff
@@ -0,0 +1,37 @@
+import { attribute, hashKey, rangeKey, table } from '@aws/dynamodb-data-mapper-annotations';
+import { CITADEL_REWARDS_DATA } from '../../config/constants';
+import { RewardEventType } from '@badger-dao/sdk/lib/citadel/enums/reward-event-type.enum';
+
+@table(CITADEL_REWARDS_DATA)
+export class CitadelRewardsSnapshot {
```

Comment:
do we need to define the indexes here?

https://github.com/Badger-Finance/badger-api/blob/0c99ec8392b46ac751d6b749d157a92e162706aa/src/aws/models/cached-boost.model.ts#L7-L22

Reply:
dunno, i'll look, we have a lot of combined indexes on the same field